send-surveys/src/test/java/com/objectcomputing/pulsesurvey/receive/responses/SurveyResponseControllerTest.java

```diff
@@ -205,4 +215,82 @@ void testMarkKeyAsUsed() {
         assertTrue(fakeResponseKey.isUsed());
     }
 
+    @Test
+    void testDisplayComments() {
+
+        String surveyKey = "123";
+
+        HttpResponse<ByteBuffer> response = httpClient
+                .exchange(HttpRequest.GET(String.format("/happiness/comment?&surveyKey=%s", surveyKey)))
+                .blockingFirst();
+
+        assertEquals(HttpStatus.OK, response.getStatus());
+
+    }
+
+    @Test
+    void testSaveUserComment() {
+
+        String fakeComments = "Here are some fake comments - Test";
+        String surveyKey =       "12345678-9123-4567-abcd-123456789abc";
+        String fakeResponseKey = "98765432-9876-9876-9876-987654321234";
+
+        UserComments fakeUserComments = new UserComments();
+        fakeUserComments.setCommentId(UUID.fromString(fakeResponseKey));
+        fakeUserComments.setResponseKey(UUID.fromString(surveyKey));
+        fakeUserComments.setCommentText(fakeComments);
+        fakeUserComments.setCreatedOn(LocalDateTime.of(2020, Month.JANUARY, 27, 1, 1));
+
+        when(mockUserCommentsRepository.save(any())).thenReturn(fakeUserComments);
+
+        assertTrue(itemUnderTest.saveUserComment(surveyKey, fakeComments));
```

Comment:
If this is why you added the boolean, then I would simply verify that the save method was called, rather than returning and asserting this.

Reply:
ok